documentation/modules/tracing/proc-instrumenting-producers-consumers-opentracing.adoc

```diff
@@ -3,15 +3,15 @@
 // assembly-instrumenting-kafka-clients-tracers.adoc
 
 [id='proc-instrumenting-producers-consumers-for-opentracing-{context}']
-= Instrumenting Kafka Producers and Consumers for tracing
+= Instrumenting producers and consumers for tracing
 
-Use a Decorator pattern or Interceptors to instrument your Java Producer and Consumer application code for distributed tracing.
+Use a Decorator pattern or Interceptors to instrument your Java producer and consumer application code for tracing.
 
 .Procedure
 
-Perform these steps in the application code of each Kafka Producer and Consumer application.
+Perform the following steps in the application code of each producer and consumer application.
```

Comment:
We wouldn't shrink as the others? So, _For each Kafka Producer and Consumer application:_

Reply:
Yes, good spot. This was feedback from the other writers.